CHANGELOG.md

```diff
@@ -126,59 +123,56 @@ This release adds support for internationalization of the module. It also contai
 - Fix issues when modifying server roles for an existing login ([MODULES-3083](https://tickets.puppet.com/browse/MODULES-3083)).
 - Fix issues when modifying an existing Windows user login ([MODULES-3752](https://tickets.puppet.com/browse/MODULES-3752)).
 
-## 2016-07-12 - Supported Release 1.1.3
-
-### Summary
-
-- Small release with several bug fixes and minor features.
-- Update the supported Puppet version ranges.
+## [1.1.3] - 2016-07-12
 
-#### Features
+### Added
 
 - Update documentation with a more advanced SQL example.
 - Add Windows Based Authentication for `sqlserver::config`. Modifies the `sqlserver::config` class with an additional property called `login_type` which can be either `SQL_LOGIN` or `WINDOWS_LOGIN`, with a default of `SQL_LOGIN`.
 
-#### Bug Fixes
+### Changed
+- Minor refactoring of code which is not used or makes code path more obvious.
+
+### Fixed
 
 - Fix Role Name Collisions. This fix introduces the database name into the title created for the `sqlserver_tsql` statements so that it is unique.
-- Minor refactoring of code which is not used or makes code path more obvious.
 - Fix TSQL error propagation. Introduce a minor refactor so that the `returns` property captures errors properly from TSQL executions.
 - Emit debug output on failed `onlyif` TSQL. Previously, there was no way of getting the log output from SQL Server when executing TSQL during an `onlyif`.
 
-## 2016-04-11 - Supported Release 1.1.2
-
-### Summary
-
-Small release to update supported Puppet version ranges.
+## [1.1.2] - 2016-04-11
 
-## 2015-12-08 - Supported Release 1.1.1
+### Changed
 
-### Summary
+- Update supported Puppet version ranges.
 
-Small release to support newer PE versions.
+## [1.1.1] - 2015-12-08
 
-## 2015-09-08 - Supported Release 1.1.0
+### Changed
```

Comment:
Is there supposed to be two changed blocks here?

Reply:
Yes, because this changed block is for 1.1.1 and the previous is for 1.1.2